nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -1241,10 +1241,16 @@ fn recursively_add_directory_to_zipfile(
 
         let file_type = entry.file_type()?;
         if file_type.is_file() {
+            let src = entry.path();
+
+            let system_mtime = entry.metadata().and_then(|m| m.modified())?;
```

Comment:
Two questions about this...

Should we ignore errors here so that we still include files that we fail to read an mtime from and continue collecting the bundle?

Since we are getting the logs in a zip file and then unpacking them on a sled where a bundle is being collected are we sure that the unzip process is preserving original modification times?